CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing Guidelines
+
+Thank you for your interest in contributing to the vCon project! We value your time and effort and are excited to have you join our community. Please read through this document to understand the guidelines and processes for contributing to this project.
+
+## Table of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+2. [Getting Started](#getting-started)
+3. [How to Contribute](#how-to-contribute)
+   - [Reporting Bugs](#reporting-bugs)
+   - [Suggesting Enhancements](#suggesting-enhancements)
+   - [Pull Requests](#pull-requests)
+4. [Style Guidelines](#style-guidelines)
+   - [Coding Style](#coding-style)
+   - [Commit Messages](#commit-messages)
+5. [Additional Resources](#additional-resources)
+
+## Code of Conduct
+
+This project and everyone participating in it are governed by our [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [ghostofbasho@gmail.com].
+
+## Getting Started
+
+To get started with contributing to this project, follow these steps:
+
+1. Fork the repository on GitHub.
+2. Clone your fork locally by running `git clone https://github.com/YOUR-USERNAME/vcon.git`.
+3. Add the original repository as a remote by running `git remote add upstream https://github.com/vcon-dev/vcon.git`.
+4. If you've added a new feature or fixed a bug, make sure to add appropriate tests.
+5. Ensure that the entire test suite passes by running the tests as specified in the project documentation.
```

Comment:
Add link [project documentation](https://github.com/vcon-dev/vcon#testing-the-vcon-package)